src/PooledArrays.jl

```diff
@@ -55,8 +57,8 @@ const PooledMatrix{T,R} = PooledArray{T,R,2}
 ##############################################################################
 
 # Echo inner constructor as an outer constructor
-function PooledArray(refs::RefArray{R}, pool::Dict{T,R}) where {T,R}
-    PooledArray{T,eltype(R),ndims(R),R}(refs, pool)
+function PooledArray(refs::RefArray{R}, revpool::Dict{T,R}) where {T,R}
+    PooledArray{T,eltype(R),ndims(R),R}(refs, revpool)
 end
 
 # A no-op constructor
```

Comment:
Actually this no-op constructor is against the current recommendataion. I think we should rather provide a `convert` method that is no-op and this constructor should provide a copy. (at least this is how I understand the current design recommendation).

(I know this is not a part of this PR but it is related - I can open a separate Issue for this)

Reply:
https://github.com/JuliaComputing/PooledArrays.jl/pull/17 :-)